src/test/java/Homework21_JRP.java

```diff
@@ -35,16 +35,13 @@ public void renamePlaylistTest() {
         WebElement editThePlaylist = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//ul/li[text()='Edit']")));
         editThePlaylist.click();
 
-        //Enter new playlist name
+        //Enter rename playlist
         WebElement newNameField = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@id='playlists']/ul/li[5]")));
```

Comment:
Try to create a locator that will work even if there is only one playlist in the "Playlists" block